charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -46,7 +46,6 @@ spec:
   template:
     metadata:
       annotations:
-        sidecar.istio.io/inject: "false"
```

Comment:
Since the earlier version of Istio only supports annotation mode, I suggest  maintaining this setting in both annotations and labels.